src/utils/network/index.ts

```diff
@@ -0,0 +1,17 @@
+import { sleep } from 'utils/common';
+
+const retrySleepTime = [2000, 5000, 10000];
+
+export async function retryAsyncFunction(func: ()=>Promise<any>, retryCount: number = 3) {
```

Comment:
What would be the behavior when someone specify `retryCount  >=3`?
Also, if am not wrong, if we call the `retryAsyncFunction` with `retryCount=1`, we would end up retrying after 10000 unit of time..

Reply:
so we never pass the retryCount and it always takes value as three, it is set so that next recursive call could be made with retryCount-1 value.

Reply:
will add a wrapper function around it to remove the confusion